spec/reek/report/location_formatter_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../../spec_helper'
+require_relative '../../../lib/reek/report/formatter'
+require_relative '../../../lib/reek/smells/duplicate_method_call'
+require_relative '../../../lib/reek/smells/smell_warning'
```

Comment:
:+1: from me – either that or a `require_lib_reek` method that would work like this:

```ruby
require_lib_reek 'smells/smell_warning'
```

Using something like a `$LIB_REEK` constant path is a bit more obvious, `require_lib_reek` a bit prettier – make a call and I’ll prepare a PR. :)